Automattic/wp-calypso
# Plugins PageView: Add is_logged_in property

#### Proposed Changes

* This PR adds the `is_logged_in` prop to the page view tracker on the `/plugins` page. This prop makes sense now since `/plugins` is also available to logged out users.

#### Testing Instructions



* Visit `/plugins` logged in and logged out and verify that the `is_logged_in` prop is correctly set on the `calypso_page_view` tracks event. To view the Tracks events firing in your browser console, visit the FG page on "How to See Calypso Tracks Events Fire in Real Time".
* Click on the browse filters like "Search Engine Optimization", "ecommerce & business" etc and verify that the prop is added to the page view event.
* Verify is_logged_in prop is added on the plugin details page view event
